api/environments/urls.py

```diff
@@ -129,4 +131,9 @@
     url(r"^", include(environments_router.urls)),
     url(r"^", include(identity_router.urls)),
     url(r"^", include(edge_identity_router.urls)),
+    path(
+        "environments/<str:environment_api_key>/edge-identities-with-identifier/edge-featurestates",
```

Comment:
I'm trying to think of a more succinct name for this but I'm finding it quite hard... 

I am wondering if we even need the additional slash though. So, could it be something like: 

```
"environments/<str:environment_api_key>/edge-identity-featurestates"
```

I guess it would perhaps be good to be somewhat explicit but if you're using `with-identifier` shouldn't we also use `featurestates-with-name` and then it just gets unwieldy.

Reply:
Yeah, that's a fair point